hbase-migration-tools/bigtable-hbase-replication/README.md

```diff
@@ -143,6 +143,38 @@ Please refer
 to [HBaseToCloudBigtableReplicationConfiguration](bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/configuration/HBaseToCloudBigtableReplicationConfiguration.java)
 for other properties that can be set.
 
+### Bidirectional Replication with Cloud Bigtable
+
+To enable bidirectional replication support with Cloud Bigtable, add the following property to `hbase-site.xml`.
+```
+<property>
+    <name>google.bigtable.replication.enable_bidirectional_replication</name>
+    <value>true</value>
+</property>
+```
+
+This feature filters out mutations replicated to Hbase from Cloud Bigtable change streams with the [Bigtable-Hbase replicator](https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/bigtable-cdc-to-hbase) to prevent replication loops. 
+
+More technically, the feature's logic checks if the last mutation of every outgoing RowMutation contains a Delete mutation on a column qualifier that matches the `cbt_qualifier` property value. If it does, that RowMutation is filtered out and not replicated.  
+
+Every replicated mutation it sends out is tagged with a Delete mutation on a qualifier set with the `hbase_qualifier` property value. The Bigtable-Hbase replicator has the same logic to filter out `hbase_qualifier` Delete mutations. 
+
+The default qualifier values are already set in [HBaseToCloudBigtableReplicationConfiguration](bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/configuration/HBaseToCloudBigtableReplicationConfiguration.java), but can also be manually set in `hbase-site.xml`.
+```
+<property>
+    <name>google.bigtable.replication.hbase_qualifier</name>
+    <value>REPLICATED_FROM_HBASE</value>
+</property>
+<property>
+    <name>google.bigtable.replication.cbt_qualifier</name>
+    <value>REPLICATED_FROM_CLOUD_BIGTABLE</value>
+</property>
+```
+
```

Comment:
Also, please add a note about the metrics being exported and why they are useful